### DISCUSSION TITLE
I2S microphone to bluetooth

### DISCUSSION THREAD

**Discussion by yo2ldk** [pschatzmann/ESP32-A2DP#362] [Q&A] [answered]
Hi,

 simple question:
 can I send audio (not file or digital sine waves) received by an I2S microphone to bluetooth, via ESP32 ?
 to use ESP as audio bluetooth transmitter, any code?
 Thank you in advance!

**Comment by pschatzmann** [comment, marked as answer]
I suggest that you have a look at my [audio-tools project](https://github.com/pschatzmann/arduino-audio-tools). 
The sketches are named logically, so look for e.g.
- basic-i2s-a2dp
- stream-i2s-a2dp
